### ISSUE TITLE
Live Preview won’t open in External Browser without using Embedded Preview first

### ISSUE THREAD

**Issue by rsgranne** [microsoft/vscode-livepreview#378]
Type: <b>Bug</b>

For at least the last year, Live Preview worked as expected:

1. Click on the Show Preview button
2. The Terminal panel would open, showing that Live Preview had started
2. The webpage I was working on would open in Brave, my default browser, & Live Preview would thereafter work as expected

However, in the last month or so, Live Preview is not working as expected.

1. Click on the Show Preview button, OR invoke the Command Palette & select “Live Preview: Show Debug Preview (External Browser)”
2. Nothing happens — the Terminal panel does *not* open, & the webpage I am working does *not* appear in Brave

I have tested this using Brave, Firefox, & Safari as my default browsers, & Live Preview fails to work in any of them.

To get Live Preview to open my webpage in Brave (or Firefox, or Safari), I have to choose one of 3 methods:

### Method 1 

1. Change Settings to go back to Embedded Preview & use that, which I prefer not to use

### Method 2

1. Leave Settings at External Browser
2. Open the Command Palette & select “Live Preview: Show Preview (Internal Browser)”, which opens my webpage in the Internal Browser (which I don’t want)
3. Click on the hamburger menu in the top right of the Internal Browser’s Preview & select Open in Browser, which then opens my webpage up in Brave
4. Go back & close the Internal Browser’s Preview tab

### Method 3

1. Leave Settings at External Browser
2. Open the Command Palette & select “Live Preview: Show Preview (Internal Browser)”, which opens my webpage in the Internal Browser (which I don’t want)
3. Close the Preview tab
4. Click on the Show Preview button, which then works correctly & opens my webpage up in Brave!

I teach a course on Web development, & all of my students reported the same problem, with some students using a Mac & others using Windows, & with a variety of Web browsers being used as defaults.

Any help or guidance appreciated.

Extension version: 0.4.3
VS Code version: Code 1.73.1 (Universal) (6261075646f055b99068d3688932416f2346dd3b, 2022-11-09T02:08:38.961Z)
OS version: Darwin arm64 22.1.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Max (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|9, 9, 9|
|Memory (System)|64.00GB (0.18GB free)|
|Process Argv|--crash-reporter-id 97ae87b2-f3a5-4847-9ee7-1a2f905967f4|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyonecf:30548226
pyindex848cf:30577861
nodejswelcome1cf:30587006
fc301958:30595537
282f8724:30602487
gswce1:30612156
3d0df643:30613357
dbltrim-noruby:30604474
f6dab269:30613381

```

</details>

**Comment by andreamah** [comment]
Hi! Is this bug still popping up with the update that was just released (0.4.4)? I realized that I missed something with the release pipeline and the update was delayed, sorry about that!

**Comment by rsgranne** [comment]
Hallelujah! I believe it is now fixed. Thank you!
